src/Webauthn.js

```diff
@@ -94,31 +94,26 @@ class Webauthn {
         return res.status(400).json({ message: 'bad request' })
       }
 
-      const user = {
-        id: base64url(crypto.randomBytes(32)),
-        [usernameField]: username,
-      }
-
-      Object.entries(this.config.userFields).forEach(([bodyKey, dbKey]) => {
-        user[dbKey] = req.body[bodyKey]
-      })
-
-      const existing = await this.store.get(username)
-      if (existing && existing.authenticator) {
-        return res.status(403).json({
-          'status': 'failed',
-          'message': `${usernameField} ${username} already exists`,
+      let user = await this.store.get(username)
```

Comment:
IIRC I think level will throw if something doesn't exist in the DB

Reply:
Looks like that's [being caught by the adapter](https://github.com/strangerlabs/webauthn/blob/master/src/LevelAdapter.js#L37-L39)